os/include/netdb.h

```diff
@@ -312,6 +312,10 @@ void endservent(void);
 
 /**
  * @brief getaddrinfo() is a function that returns information on a particular host name (such as its IP address) and loads up a struct sockaddr.
+ *
+ * @details @b #include <netdb.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised